pkg/output/output.go

```diff
@@ -210,6 +216,30 @@ func (b *Handler) writeSIGRetestBadge(name, filePath string, data types.RunningA
 	return badge.Render(name, badgeString, color, f)
 }
 
+func (b *Handler) writeJobFailureBadges(data types.RunningAverageDataItem, levels *Levels) error {
+	failedJobLeaders := data.FailedJobLeaderBoard
+	basePath, err := b.initializeSourcePath()
+	if err != nil {
+		return err
+	}
+	for i, job := range failedJobLeaders {
+		if i < 10 {
```

Comment:
Thanks!

Algorithm-wise, I feel a bit uneasy with staying in the loop after 10 iterations. How about `break`ing on 10?

Reply:
Yes of course - makes sense.